app/app.js

```diff
@@ -1,15 +1,48 @@
+
 const express = require('express');
 const bodyParser = require('body-parser');
 const logger = require('morgan');
 const userRouter = require('./routes/userRouter.js');
+const request = require('superagent');
 
 const app = express();
 
+require('dotenv').config();
+process.env.ENV = process.env.ENV || 'dev';
+
+
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
 app.use(logger('dev'));
 
 app.use('/api/users', userRouter);
 
+////ANNIE'S API CALLS BEGIN HERE///////////////////////////////////////////
+const apiKey = process.env.API;
+const movieId = 550;
+
+/// ONCE WE ALREADY HAVE THE movieId (above)
+// GET DETAILS BASED ON movieId
+app.get('/', (req, res) => {
```

Comment:
Can you rename this route to `'/api/trailers/:id` or something similar? That way, we can start building out the API.